doc/zh-cn/README.md

```diff
@@ -69,3 +69,17 @@
 * [authz-keycloak](plugins/authz-keycloak-cn.md): 支持 Keycloak 身份认证服务器
 * [uri-blocker](plugins/uri-blocker.md): 根据 URI 拦截用户请求。
 * [oauth](plugins/oauth.md): 提供 OAuth 2 身份验证和自省。
+
+部署
```

Comment:
better also add the English verison

Reply:
English version already available in https://github.com/apache/incubator-apisix/blob/master/doc/README.md